### PR TITLE
Hotel CRUD

### PR DESCRIPTION
**Pull request by chenyirex**
Added phone_number as a column in hotel schema

### CODE REVIEW

File: src/routes/hotels.php [hunk 1]
```diff
@@ -6,28 +6,236 @@
...
+            return $response->withStatus(500);
+        }
+    } catch (PDOException $e) {
+        $db->rollBack();
+        $db = null;
+
+        $response->write($e);
+        return $response->withStatus(500);
+    }
+});
```
Comment (suiyoubi) [inline src/routes/hotels.php]: if you dont want to set the db to null several times, you could set it here and return the $response.

File: src/routes/hotels.php [hunk 2]
```diff
@@ -6,28 +6,236 @@
...
+                $db = null;
+
+                $response->write('{"error": {"text": "failed create on hotel"}}');
+                return $response->withStatus(500);
+            }
+        } else {
+            $db = null;
+
+            $response->write('{"error": {"text": "failed create on address"}}');
+            return $response->withStatus(500);
```
Comment (suiyoubi) [inline src/routes/hotels.php]: you can only set the status code `$response->withStatus(500)` and return at the end of the function

File: src/routes/hotels.php (hunk 1 above)
Comment (suiyoubi) [reply src/routes/hotels.php]: `$db = null; return $response`

Comment (chenyirex) [reply src/routes/hotels.php]: make sense

**Pull request merged**